Zilon.Core/Zilon.Core.Specs/Contexts/FeatureContextBase.cs

```diff
@@ -113,45 +99,34 @@ public async Task CreateSectorAsync(int mapSize)
                 return Task.FromResult(map);
             });
 
-            var sectorManager = ServiceProvider.GetRequiredService<ISectorManager>();
-            var humanPlayer = ServiceProvider.GetRequiredService<HumanPlayer>();
+            var globeInitialzer = ServiceProvider.GetRequiredService<IGlobeInitializer>();
+            var globe = await globeInitialzer.CreateGlobeAsync("intro").ConfigureAwait(false);
+            Globe = globe;
 
-            var sectorSubScheme = new TestSectorSubScheme
-            {
-                RegularMonsterSids = new[] { "rat" },
-                MapGeneratorOptions = new SquareGenerationOptionsSubScheme
-                {
-                    Size = mapSize
-                }
-            };
+            var humanPlayer = ServiceProvider.GetRequiredService<IPlayer>();
 
-            var sectorNodeMock = new Mock<ISectorNode>();
-            sectorNodeMock.SetupGet(x => x.State).Returns(SectorNodeState.SectorMaterialized);
-            sectorNodeMock.SetupGet(x => x.SectorScheme).Returns(sectorSubScheme);
-            var sectorNode = sectorNodeMock.Object;
+            var sectorNode = globe.SectorNodes.First();
 
             humanPlayer.BindSectorNode(sectorNode);
-
-            await sectorManager.CreateSectorAsync();
         }
 
-        private class SquareGenerationOptionsSubScheme : ISectorSquareMapFactoryOptionsSubScheme
-        {
-            public SchemeSectorMapGenerator MapGenerator { get => SchemeSectorMapGenerator.SquarePlane; }
-            public int Size { get; set; }
-        }
+        //private class SquareGenerationOptionsSubScheme : ISectorSquareMapFactoryOptionsSubScheme
+        //{
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=6105716)